compiler+runtime/test/bash/clojure-test-suite/src/jank_test/run_clojure_test_suite.cljc

```diff
@@ -0,0 +1,137 @@
+(ns jank-test.run-clojure-test-suite
+  (:require [clojure.test :as t]))
+
+;; most of these are blocked by parsing of #?(:clj Long/MAX_VALUE)
+;; https://github.com/jank-lang/jank/issues/195
+(def namespaces
```

Comment:
I really appreciate the rigor put into this.

Reply:
Thanks for saying that :)